### PR TITLE
Add test

### PR DESCRIPTION
**Pull request by Joshua-onwuzu**
- [ ] Bug fix 
- [ ] New feature 
- [x] Chore 
- [ ] Breaking change 
- [ ] There is an open issue which this change addresses
- [ ] My commits follow the [Git Commit Guidelines](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#commits)
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly.
- [ ] My change requires a change to Typescript typings.
  - [ ] I have updated the typings accordingly.

### CODE REVIEW

**Review by nadeem-fileverse — CHANGES_REQUESTED** [review]
PTAL

File: packages/heartbit-core/src/test/mockData.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+export const mintData = {
+    message: "p6f7gr-5173.csb.app wants you to sign in with your Ethereum account:\n0x852bd84A4dcDc150648AE4e8D6D6e59b1797c86f\n\nHello World!\n\nURI: https://p6f7gr-5173.csb.app\nVersion: 1\nChain ID: undefined\nNonce: 6RnPMnIbRS1yQ02ZK\nIssued At: 2024-02-15T09:14:18.897Z",
+    signature: "0xfff4ce944d488abf2f8b2fd80bf3f585cbd14bbafe37ffd8b88e69ad88bd0c86426898416eb2c9c4f913eaa01430f4cac405c5a5c53a6410652b56874b9bd45e1c",
+    startTime: 1706898250,
+    endTime: 1706898251,
+    hash: "0xba0c379a9b364b41e69a6a00a8652e28cb7dda3ca684309b940807634919a940"
+  };
+export const unsignedMintData = {
+    message: "p6f7gr-5173.csb.app wants you to sign in with your Ethereum account:\n0x852bd84A4dcDc150648AE4e8D6D6e59b1797c86f\n\nHello World!\n\nURI: https://p6f7gr-5173.csb.app\nVersion: 1\nChain ID: undefined\nNonce: 6RnPMnIbRS1yQ02ZK\nIssued At: 2024-02-15T09:14:18.897Z",
```
[the remark this answers is not in the document]
Comment (Joshua-onwuzu) [reply packages/heartbit-core/src/test/mockData.ts]: done

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+      - name: 🔆 Cache pnpm modules
+        uses: actions/cache@v3
+        with:
+          path: ~/.pnpm-store
+          key: ${{ runner.os }}-pnpm-${{ hashFiles('**/pnpm-lock.yaml') }}
+          restore-keys: |
+            ${{ runner.os }}-pnpm-
+
+      - name: 🧩 Install Dependencies
```
Comment (nadeem-fileverse) [inline .github/workflows/test.yml:53]: build can be made from the root command i think check for build command in root package.json

File: .github/workflows/test.yml [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+        run: pnpm install
+        working-directory: ./packages/heartbit-core
+
+      - name: 🏗️ Build
+        run: pnpm run build
+        working-directory: ./packages/heartbit-core
+
+      - name: 🧪 Run Tests
```
Comment (nadeem-fileverse) [inline .github/workflows/test.yml:61]: update for heartbit react

**Review by nadeem-fileverse — CHANGES_REQUESTED** [review]
PTAL

**Pull request closed without merging**